test/test_default_api.py

```diff
@@ -903,6 +903,283 @@ def test_update_an_incoming_number(self):
+        api_response.next_page_uri = "/Accounts/{accountId}/IncomingPhoneNumbers?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, IncomingNumberList)
+
+    def  test_list_participants_get_next_page(self):
+        """Test case for get_next_page using list_participants response
+
+        Get Next Page for List Participants  # noqa: E501
+        """
+
+        conference_id = conference_id_list_participants_test_value
+        talk = talk_list_participants_test_value
+        listen = listen_list_participants_test_value
+        
+        api_response = self.api.list_participants(conference_id=conference_id,talk=talk,listen=listen)
+        api_response.next_page_uri = "/Accounts/{accountId}/Conferences/{conferenceId}/Participants?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, ConferenceParticipantList)
+
+    def  test_list_recordings_get_next_page(self):
+        """Test case for get_next_page using list_recordings response
+
+        Get Next Page for List Recordings  # noqa: E501
+        """
+
+        call_id = call_id_list_recordings_test_value
+        conference_id = conference_id_list_recordings_test_value
+        date_created = date_created_list_recordings_test_value
+        
+        api_response = self.api.list_recordings(call_id=call_id,conference_id=conference_id,date_created=date_created)
+        api_response.next_page_uri = "/Accounts/{accountId}/Recordings?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, RecordingList)
+
+    def  test_list_sms_messages_get_next_page(self):
+        """Test case for get_next_page using list_sms_messages response
+
+        Get Next Page for List SMS Messages  # noqa: E501
+        """
+
+        to = to_list_sms_messages_test_value
+        _from = _from_list_sms_messages_test_value
+        begin_time = begin_time_list_sms_messages_test_value
+        end_time = end_time_list_sms_messages_test_value
+        direction = direction_list_sms_messages_test_value
+        campaign_id = campaign_id_list_sms_messages_test_value
+        brand_id = brand_id_list_sms_messages_test_value
+        is10_dlc = is10_dlc_list_sms_messages_test_value
+        
+        api_response = self.api.list_sms_messages(to=to,_from=_from,begin_time=begin_time,end_time=end_time,direction=direction,campaign_id=campaign_id,brand_id=brand_id,is10_dlc=is10_dlc)
+        api_response.next_page_uri = "/Accounts/{accountId}/Messages?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, MessagesList)
+
+    def  test_make_a_call_get_next_page(self):
```

Comment:
does make_a_call have a paginated response?

Reply:
good catch, added x-is-paginated to make-a-call instead of list-calls in the openapi spec. should be updated now